test/karma.custom-launchers.config.js

```diff
@@ -140,8 +16,27 @@ const customLaunchers = {
 const getAllBrowsers = () => Object.keys(customLaunchers);
 const getRandomBrowser = () => sample(getAllBrowsers());
 
+/**
+ * Environment variables are passed into the script and the depth of testing
+ * is affected accordginly.
+ *
+ * - Whenever on a PR we only want to probe test with Firefox
+ * - Whenever we are on the most recent node version on Travis we test via BrowserStack
+ * - If none of the prior mentioned holds we assume to be running local and respect the passed
+ *   in borwsers argv
+ */
+const shouldProbeOnly = argv.shouldProbeOnly !== '';
+const shouldTestOnBrowserStack = argv.shouldTestOnBrowserStack !== '';
+const defaultBrowsers = ['Firefox'];
+const argvBrowsers = isArray(argv.browsers)
```

Comment:
Defaulting to FF
Respecting whatever we get passed in via argv

If we want BS and we probe we run FF otherwise any browser
If we don't we just pick from argv with the same defaulting to FF